src/sentry/migrations/0254_org_integration_grace_period_end.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.24 on 2021-11-29 21:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0253_add_widget_type"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="organizationintegration",
+            name="grace_period_end",
+            field=models.DateTimeField(blank=True, null=True),
```

Comment:
Will you be using this field in conditions for the proposed 15 min celery job? If so you should consider putting an index on this column so that you don't have to table scan.

Reply:
@markstory Yes, I plan on using that in the query. Guess I can update this PR with that change, yea?